### PR TITLE
Webhook URL extension for every type of webhook

### PR DESCRIPTION
**Pull request by temp666**
A few webhooks only need raids and other need raids and mons.
Here is the extension that it is possible to enter your endpoint for each type of webhook.

Defaults to everything activated:
`webhook_url=http://example.org/foo/bar`

To activate certain options:
`webhook_url=[mon|weather|raid]http://example.org/foo/bar`

With multiple urls:
`webhook_url=[mon|weather]http://example.org/foo/bar,[raid]http://othersite.com/baz`

### CODE REVIEW

**Comment by sn0opy** [comment]
Looks good to me. Thanks!

File: utils/walkerArgs.py [hunk 1]
```diff
@@ -152,7 +152,7 @@ def parseArgs():
     parser.add_argument('-wh', '--webhook', action='store_true', default=False,
                         help='Activate webhook support')
     parser.add_argument('-whurl', '--webhook_url', default='',
```
Comment (fosJoddie) [inline utils/walkerArgs.py]: I think you can make multiple webhook urls easier to setup if you add ` action='append'` to the parser argument definition.

Then you can simply do: 

```
webhook_url: [mon|weather|raid]http://example.org/foo/bar 
webhook_url: [raid]http://otherUrl.org/foo/bar 
```
for a less cluttered way of configuration.

Comment (sn0opy) [reply utils/walkerArgs.py]: I like the idea, thing is: what's easier to explain? The extra line or the comma seperated list?

Comment (fosJoddie) [reply utils/walkerArgs.py]: I'd say it's easier to just add lines, the users who need more than one would probably be techy enough to figure it out any which way

Comment (sn0opy) [reply utils/walkerArgs.py]: Okay, aggreed. @temp666 if you like, feel free to PR this as well, otherweise I'm going to work on it next week.

**Pull request merged**